cacheops/lua/invalidate.lua

```diff
@@ -12,31 +12,18 @@ local conj_cache_key = function (db_table, scheme, obj)
     return 'conj:' .. db_table .. ':' .. table.concat(parts, '&')
 end
 
-local call_in_chunks = function (command, args)
-    local step = 1000
-    for i = 1, #args, step do
-        redis.call(command, unpack(args, i, math.min(i + step - 1, #args)))
-    end
+local rename_key = function (old_key, new_key)
+    redis.call('rename', old_key, new_key)
 end
 
-
 -- Calculate conj keys
-local conj_keys = {}
+local renamed_keys = {}
 local schemes = redis.call('smembers', 'schemes:' .. db_table)
 for _, scheme in ipairs(schemes) do
-    table.insert(conj_keys, conj_cache_key(db_table, scheme, obj))
+    local key = conj_cache_key(db_table, scheme, obj)
+    local new_key = key .. '.delete'
+    pcall(rename_key, key, new_key)
```

Comment:
`pcall(...)` in Lua means to make the call and return a second `status` return value of `0` if there were any exceptions. If `key` doesn't exist because no resultsets have been cached for this `conj` key, we just want to ignore that exception.